README.mdown

```diff
@@ -1,11 +1,11 @@
 # Overview
 
-This is a SketchUp extension for exporting FBX models to be used with
+This is a SketchUp extension for exporting FBX or Collada models to be used with
 Colossal Order's Cities: Skylines.
 
 Cities: Skylines uses the Unity engine which require assets to have a flat
 model hierarchy. Unfortunately SketchUp's FBX exporter adds an extra node even
-if all the faces are in the root of the model.
+if all the faces are in the root of the model. 
```

Comment:
Some extra trailing white-space here. What editor do you use? Normally there are settings to automatically remove these. (The Sublime Text project is set up for this already.)